### PR TITLE
Dialing down gerrit auth noise: log only when cookiefile changed

### PR DESCRIPTION
**Pull request by chaodaiG**
This is now very flaky, logs every couple of seconds

/cc @cjwagner @listx

### CODE REVIEW

File: prow/gerrit/client/client.go [hunk 1]
```diff
@@ -319,12 +320,11 @@ func (c *Client) Authenticate(cookiefilePath, tokenPath string) {
 	c.lock.Lock()
 	was, c.authentication = c.authentication, auth
 	c.lock.Unlock()
-	logrus.Info("Authenticating gerrit requests...")
```
Comment (listx) [inline prow/gerrit/client/client.go:322]: Nit: It would be much cleaner to separate this out into its own commit. The way it stands it's hard to understand the diff vs the commit message.

**Review by listx — COMMENTED** [review]
/lgtm
/hold

File: prow/gerrit/client/client.go (hunk 1 above)
Comment (chaodaiG) [reply prow/gerrit/client/client.go:322]: ack
